source/locales/es/LC_MESSAGES/contents.po

```diff
@@ -0,0 +1,19 @@
+# SOME DESCRIPTIVE TITLE.
+# Copyright (C) 2021, The Natural Capital Project
+# This file is distributed under the same license as the InVEST package.
+# FIRST AUTHOR <EMAIL@ADDRESS>, 2021.
```

Comment:
Is it possible to skip `contents.rst`? I don't think there's anything to translate.

Reply:
`sphinx-intl` doesn't seem to have any option to skip a file - it makes one PO file for each RST file in a given directory. And this file will get re-created whenever we do `sphinx-intl update`. We could delete it now and each future time, but I think it's also okay to keep it here. No strong opinion.

Reply:
I think it's fine to leave as is.